code/game/machinery/doors/passworddoor.dm

```diff
@@ -12,6 +12,9 @@
 	var/password = "Swordfish"
 	var/interaction_activated = TRUE //use the door to enter the password
 	var/voice_activated = FALSE //Say the password nearby to open the door.
+	var/doorOpen = 'sound/machines/blastdoor.ogg' //Sound used upon opening.
+	var/doorClose = 'sound/machines/blastdoor.ogg' //Sound used upon closing.
+	var/doorDeny = 'sound/machines/buzz-sigh.ogg' //Sound used upon denying.
```

Comment:
Should be in snake_case

Reply:
Curious... airlock.dm (used as a reference) had camelCase everywhere...